templates/Default/engine/Default/feature_request.php

```diff
@@ -74,11 +80,11 @@
 				<td align="center"><textarea spellcheck="true" name="feature" id="InputFields" maxlength="500"></textarea></td>
 			</tr>
 			<tr>
-				<td align="center">Anonymous: <input name="anon" id="InputFields" type="checkbox" checked="checked"/></td>
+				<td align="center">Anonymous: <input name="anon" id="InputFields" type="checkbox" /></td>
```

Comment:
I'd rather keep this as default anonymous, the idea is to allow people to safely request features and changes without having to worry about the idea being rejected due to the person who requested it rather than due to the idea's merits or lack thereof , and without having to worry about being harassed by other players in the game about it (which can and does happen, unless the playerbase has massively changed in the past few years).  And the reason it needs to be the default rather than opt-in is because if you forgot to tick it you can't then make it anonymous, however you can then make a comment claiming it as yours

Reply:
Fair enough! I'll revert that commit now.